pkg/render/render.go

```diff
@@ -191,7 +191,6 @@ type calicoRenderer struct {
 func (r calicoRenderer) Render() []Component {
 	var components []Component
 	components = appendNotNil(components, PriorityClassDefinitions(r.installation))
-	components = appendNotNil(components, Namespaces(r.installation, r.provider == operator.ProviderOpenShift, r.pullSecrets))
```

Comment:
This won't work because the namespace needs to exist for the ConfigMaps, Secrets, and Typha components, so moving it to the "Node" component will not work.